README.md

```diff
@@ -200,5 +200,5 @@ configuring, initializing, and starting your application.
 [ll]: https://github.com/apache/fluo-uno/blob/master/LICENSE
 [logo]: contrib/uno-logo.png
 [Muchos]: https://github.com/apache/fluo-muchos
-[ssh-docs]: https://hadoop.apache.org/docs/r2.7.2/hadoop-project-dist/hadoop-common/SingleCluster.html#Setup_passphraseless_ssh
+[ssh-docs]: https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/SingleCluster.html#Setup_passphraseless_ssh
```

Comment:
"current" is a moving target. If we just want to update to the latest, where we know the information still exists (it could be moved in newer "current" versions), I suggest using `r3.2.1` instead.